annotation/src/main/java/org/cbioportal/genome_nexus/annotation/web/AnnotationController.java

```diff
@@ -218,6 +225,47 @@ public AnnotationController(VariantAnnotationService variantAnnotationService,
         return getHotspotAnnotation(variants);
     }
 
+    @ApiOperation(value = "Retrieves mutation assessor information for the provided list of variants",
+        nickname = "getMutationAssessorAnnotation")
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = "Success",
+            response = MutationAssessor.class,
+            responseContainer = "List"),
+        @ApiResponse(code = 400, message = "Bad Request")
+    })
+    @RequestMapping(value = "/mutation_assessor/{variants:.+}",
+        method = RequestMethod.GET,
+        produces = "application/json")
+    public List<MutationAssessor> getMutationAssessorAnnotation(
+        @PathVariable
+        @ApiParam(value="Comma separated list of variants. For example 7:g.140453136A>T,12:g.25398285C>A",
+            required = true,
+            allowMultiple = true)
+            List<String> variants)
+    {
+        List<MutationAssessor> mutationAssessors = new ArrayList<>();
+        for (String variant : variants)
+        {
+            mutationAssessors.add(getMutationAnnotation(variant));
+        }
+        return mutationAssessors;
+    }
+
+    @ApiOperation(value = "Retrieves mutation assessor information for provided list of variants",
+        nickname = "postMutationAssessorAnnotation")
+    @RequestMapping(value = "/mutation_assessor",
+        method = RequestMethod.POST,
+        produces = "application/json")
+    public List<MutationAssessor> postMutationAssessorAnnotation(
+        @RequestParam
+        @ApiParam(value="Comma separated list of variants. For example 7:g.140453136A>T,12:g.25398285C>A",
+            required = true,
+            allowMultiple = true)
+        List<String> variants)
```

Comment:
Using a list parameter is okay. But getting the list as a comma separated `query` or `path` parameter is suboptimal. We need to change all of these to `body` parameters at some point. A very long `path` or `query` parameter may potentially break the query. AFAIK, in cBioPortal API, list parameters are defined as `body` parameters.